lib/package.json

```diff
@@ -9,22 +9,21 @@
     "url": "https://dxc.com"
   },
   "main": "./dist/main.js",
+  "types": "./dist/main.d.ts",
   "license": "Apache-2.0",
   "peerDependencies": {
     "react": "^18.x",
     "react-dom": "^18.x",
-    "@dxc-technology/halstack-react": "6.x || 0.0.0-x"
+    "@dxc-technology/halstack-react": "8.0.0"
```

Comment:
We should set the version to "^8.0.0" just for compatibility reasons.

Reply:
Done!